### DISCUSSION TITLE
Block outbound emails to unknown recipients (when no reverse alias exists)?

### DISCUSSION THREAD

**Discussion by argarner** [simple-login/app#1305] [Feature Requests]
My email was exposed when I replied to an old email, after updating the recipient to a newly created reverse alias, because there was a CC I didn't see (hidden by my email client). The company then replied from the CC address directly to my real email.

I'd like to be able to enable the universal blocking/rejection of outbound emails (from all of my mailboxes, or individually) to all "unknown" recipients (when no reverse alias exists), to prevent this from happening in future.

**Comment by nguyenkims** [comment]
@argarner unfortunately in this case the email wasn't sent to SimpleLogin server and we won't be able to alert you ...

**Reply by argarner** [reply]
Hahaha @nguyenkims. Yes, ofcourse! Derp.

I only raised the issue as I remembered receiving a SimpleLogin email "Email sent contains non reverse-alias addresses", but I now realise that was just a heads up. Thanks for the warning, anyway! Cheers!
